src/components/calcite-pick-list/calcite-pick-list.scss

```diff
@@ -1,24 +1,17 @@
 :host {
   @extend %component-host;
-  @extend %component-spacing;
-  align-items: stretch;
-  background-color: transparent;
-  display: flex;
+  @apply items-stretch bg-transparent flex flex-col relative pb-0;
   flex: 1 0 auto;
-  flex-flow: column;
-  padding-bottom: var(--calcite-cap-spacing);
-  position: relative;
 }
 
 header {
-  background-color: var(--calcite-ui-foreground-1);
-  display: flex;
-  justify-content: flex-end;
-  align-items: stretch;
-  margin-bottom: var(--calcite-spacing-quarter);
+  @apply bg-foreground-1 
+    flex 
+    justify-end 
+    items-stretch;
+  margin-bottom: theme("spacing.1");
   &.sticky {
-    position: sticky;
-    top: 0;
+    @apply sticky top-0;
     z-index: 1;
```

Comment:
Out of scope for this pr but it would be good if we could come up with a strategy for the z- utils.

Reply:
sounds good.